.github/workflows/run-pest-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: Run Pest tests
+
+on:
+  workflow_call:
+    inputs:
+      php-versions:
+        description: 'Json array of PHP versions'
+        default: '["8.1","8.2"]'
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 5
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: ${{ fromJSON(inputs.php-versions) }}
+
+    name: PHP${{ matrix.php }} - Laravel${{ matrix.laravel }}
+
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick, fileinfo, xdebug
+          coverage: xdebug
+          tools: composer:v2
+        env:
+          COMPOSER_TOKEN: ${{ secrets.YARD_BOT_PAT }}
+
+      - name: Setup problem matchers
+        run: |
+          echo "::add-matcher::${{ runner.tool_cache }}/php.json"
+          echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+      - name: Install dependencies
+        run: |
+          composer install
```

Comment:
Is dit correct? De Laravel test runner gebruikt composer require / update commands hier.

Reply:
Als je de tests tegen verschillende php versies wil kunnen testen moet je wel composer update doen.